src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -537,7 +538,7 @@ export function useSwapInputsController({
   };
 
   const quoteFetchingInterval = useAnimatedInterval({
-    intervalMs: 12_000,
+    intervalMs: IS_TEST ? 999_999 : 12_000,
```

Comment:
Wouldn't this be a problem and cause flakiness whenever if the quote is no longer valid?

Reply:
potentially yeah. i guess the only fix would be to mock the quote then?

Reply:
I can approve after you roll back that change!